src/Whoops.php

```diff
@@ -136,25 +136,37 @@ protected function createWhoopsInstance(ServerRequestInterface $request): Run
 
     private static function shouldUpdateResponse(Run $whoops): bool
     {
-        if (1 !== count($whoops->getHandlers())) {
+        $handlers = $whoops->getHandlers();
+        if (count($handlers) === 0) {
             return false;
         }
 
-        $handler = current($whoops->getHandlers());
+        $plainTextWithLoggerOnly = array_filter($handlers, function ($handler) {
+            return $handler instanceof PlainTextHandler && $handler->loggerOnly();
+        });
 
-        return !($handler instanceof PlainTextHandler && $handler->loggerOnly());
+        if ($plainTextWithLoggerOnly && count($plainTextWithLoggerOnly) === count($handlers)) {
```

Comment:
If I understand correctly, you try to find out if there is a PlainTextHandler present in the handlers. I don't understand what the `->isLoggerOnly()` does. 

That said, I think this method can be somewhat simpler. The `count($handlers) === 0`  block can be removed.
After the array filter you can just do `return count($plainTextWithLoggerOnly) > 0;`

Reply:
`loggerOnly` makes the PlainTextHandler work as a logger only. So it logs but it doesn't output anything.

Reply:
@storeman just removed the === 0 condition - check it and I do the release